### ISSUE TITLE
Python files

### ISSUE THREAD

**Issue by sailor88128** [HadXu/machine-learning#1]
Is this based on tensorflow? I didnot find the py file in cloning files.

**Comment by HadXu** [comment]
No,this is based on theano and keras,and i use jupyter notebook,so you can run jupyter notebook on cmd
